File: API_Escola_Stable/Models/Aula.cs
using System.ComponentModel.DataAnnotations;
//FELIPE
namespace API_Uni_Func.Models
{
    public class Aula
    {
        [Key]
        public int IdAula { get; set; }
        public int IdDisciplina {get; set; }
        public Disciplina? DiciplinaDoDia { get; set; }
        public DateTime DataDiciplina { get; set; }
        public int IdSala {get; set; }
        public Sala? SalaDeAula { get; set; }

        public int CronogramaId { get; set; }
        public Cronograma? Cronograma {get; set;} 
    }
}

File: AppUniversidade/src/app/Aula.ts
import { Disciplina } from "./Disciplina";
import { Sala } from "./Sala";
import { Cronograma } from "./Cronograma";

export class Aula{
    IdAula: number = 0;
    DiciplinaDoDia: Disciplina | undefined;
    DataDiciplina: Date | undefined;
    SalaDeAula: Sala | undefined;

    CronogramaId: number = 0;
    Cronograma: Cronograma | undefined;
}